src/main/resources/en/elden_ring.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  faker:
+    elden_ring:
+      location: ["Limgrave","Weeping Penisula", "Liurnia Of The Lakes","Caelid","Altus Plateau","MT.Gelmir","Dragonbarrow","Mountaintops Of The Giants","Consecrated Snowfield"]
+      weapon: ["Dagger","Cinquedea","Short Sword","Sword of Night and Flame","Claymore","Dark Moon Greatsword","Grafted Blade Greatsword","Rapier","Carian Glintstone Staff"]
+      skill: ["Lion's Claw","Impaling Thrust","Piercing Fang","Spinning Slash","Charge Forth","Stamp","Blood Tax","Repeating Thrust","Wild Strikes","Spinning Strikes","Double Slash","Prelate's Charge"]
+      spell: ["Adula's Moonblade","Cannon Of Haima","Collapsing Stars","Comet","Glintstone Cometshard","Great Oracular Bubble","Loretta's Mastery","Ranni's Dark Moon"]
+      NPC: ["Sorceress Sellen","Smithing Master Hewg","Sorcerer Rogier","D","Rennala","Renna","Fia","Irina","Queen Marika"]
```

Comment:
Lowercase here too please

Reply:
Do we need to lower all the letters include names?